zstacklib/zstacklib/utils/sanlock.py

```
@@ -515,3 +666,77 @@ def get_watchdog_fire_timeout():
+    lease.sector_size = get_sector_size(vg_name)
+    lease.align_size = sector_size_to_align_size(lease.sector_size)
+    if lease.resource_name == "GLLK":
+        lease.offset = GLLK_BEGIN * lease.align_size
+    elif lease.resource_name == "VGLK":
+        lease.offset = VGLK_BEGIN * lease.align_size
+    else:
+        # LV lease
+        from zstacklib.utils import lvm
+        lease.offset = lvm.get_lv_lockargs(os.path.join("/dev/", vg_name, resource_name))
+
+    return request_repair_metadata(vg_name, **lease.__dict__)
```

Pylint (3.3.7)

[error] Argument 'vg_name' passed by position and keyword in function call

(E1124)